modules/lib/src/main/resources/assets/styles/browse/content-app-panel.less

```diff
@@ -1,3 +1,3 @@
 .content-app-panel {
-  top: 44px;
+  /* top: 44px !important; */
```

Comment:
Remove the file?

Reply:
Will do

Reply:
Regarding blinking: ContentAppBar knows nothing about loaded widgets, and it is a responsibility of a widget to hide/show viewer on demand when they get loaded. However I can remove initial blink by not showing viewer and app name until first widget gets selected.

Reply:
Removed extra blink until first widget selected. Further improvements will require additional investigation and refactoring